src/main/webapp/assets/modules/get-promoted/gp-form.mod.js

```diff
@@ -0,0 +1,34 @@
+techlooper.directive("getPromotedForm", function ($http, vnwConfigService, $location) {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/get-promoted/gp-form.tem.html",
+    link: function (scope, element, attr, ctrl) {
+      var campaign = $location.search();
+
+      scope.doPromotion = function () {
+        if (!scope.promotionForm.$valid) {
+          return false;
+        }
+        scope.masterPromotion = angular.copy(scope.promotionInfo);
+        $http.post("getPromoted", scope.promotionInfo)
+          .success(function (data, status, headers, config) {
```

Comment:
'config' is defined but never used.<br>'headers' is defined but never used.<br>'status' is defined but never used.